content/blog/2023-01-wazo-chatgpt.md

```diff
@@ -0,0 +1,156 @@
+---
+title: Hey ChatGPT, write an article about Wazo and you
+date: 2023-01-11 10:00:00
+author: Sylvain BOILY
+category: Wazo Platform
+tags: hack, chat, websocket, chatgpt, openai
+slug: 2023-chatgpt-with-wazo
+status: published
+---
+
+## ChatGPT, chatgpt
+
+Hey my new friend, please i need you for my creativity. If you don't know what's is chatGPT yet
+please look on your social network, internet etc... because this is the latest big trend for 2023.
+ChatGPT could be everywhere in everything and this could probably be, the tech world next huge company.
```

Comment:
```suggestion
ChatGPT could be everywhere in everything, and this could probably be the tech world's next huge company.
```

Reply:
not sure on the meaning of "next huge company" here. Is this refering to OpenAI(the company behind ChatGPT)? Perhaps you meant "next huge gamechanger", "next huge revolution", "next huge paradigm shift" or similar.